packages/config/src/index.js

```diff
@@ -23,12 +24,14 @@ const resolveConfig = async function(configFile, { cwd, ...options } = {}) {
   return configC
 }
 
-const getConfig = function(configPath, options) {
+const getConfig = async function(configPath, options) {
   if (configPath === undefined) {
     return {}
   }
 
-  return configorama(configPath, {
+  const config = await parseConfig(configPath)
+
+  return configorama(config, {
```

Comment:
Still using configorama?

Reply:
Yes I will remove its support in a follow-up PR. This one is just to separate parsing.